tooling/Slack-channels.md

```diff
@@ -0,0 +1,17 @@
+##Slack
+
+##Join slack: broadinstitute.slack.com
+
+Slack is a major communication method for the team for questions and discussion. Saturn-team is the primary team channel.
+
+Recommended channels: 
+
+* Saturn-team
+* Saturn
+* dsp-engineering
+* dsp-workbench
+* pets
+* prometheus
+* softeng
+* 105b
+* social
```

Comment:
Optional: It may be worth adding `workbench-userliaison` to this list? (I mention it in the User Liaison doc, so, probably not necessary since you should only need to watch the channel if you are in that role.)

Reply:
That makes sense, I'll add it! I put in a quick description as well, but you should take a look to make sure it's accurate/complete since I'm not too familiar with the user liaison procedure.

Reply:
If you're adding `workbench-userliaison` I would recommend maybe adding like a "priveleged" or "extra" section and include that and also the on-call channels ( I will get those for you in a sec)

Reply:
Maybe I should break it out into sections, like 'team/work channels' 'social channels' 'extra channels' (I'm not sure 'extra' is the best descriptor but I can't think of what else to call it).

Reply:
yes I like that idea !